L/LLVM/common.jl

```diff
@@ -18,6 +18,7 @@ const llvm_tags = Dict(
     v"15.0.7" => "2593167b92dd2d27849e8bc331db2072a9b4bd7f", # julia-15.0.7-10
     v"16.0.6" => "499f87882a4ba1837ec12a280478cf4cb0d2753d", # julia-16.0.6-2
     v"17.0.6" => "0424864b3ac348210aab171668eea1a669956a31", # julia-17.0.6-3
+    v"18.1.5" => "6f748bf1d110dbf9a4ab7ff6dac261c64865d514", # julia-18.1.5-0
```

Comment:
Reminder to self: we still need to create the tag, we'll do it when we're reasonably sure we've got all the patches we need.

Reply:
Tag created: https://github.com/JuliaLang/llvm-project/releases/tag/julia-18.1.5-0